api/src/identity-access-management/scripts/import-user-last-logged-at.script.js

```diff
@@ -0,0 +1,61 @@
+import Joi from 'joi';
+
+import { csvFileParser } from '../../shared/application/scripts/parsers.js';
+import { Script } from '../../shared/application/scripts/script.js';
+import { ScriptRunner } from '../../shared/application/scripts/script-runner.js';
+import { usecases } from '../domain/usecases/index.js';
+
+export const csvSchemas = [
+  { name: 'userId', schema: Joi.number().required() },
+  { name: 'last_activity', schema: Joi.date().required() },
+];
+
+export class ImportUserLastLogeedAtScript extends Script {
+  constructor() {
+    super({
+      description: 'This script allows to update user last logged at',
+      permanent: false,
+      options: {
+        file: {
+          type: 'string',
+          describe: 'CSV file path',
+          demandOption: true,
+          coerce: csvFileParser(csvSchemas),
+        },
+        dryRun: {
+          type: 'boolean',
+          describe: 'Executes the script in dry run mode',
+          default: false,
+        },
+      },
+    });
+  }
+  async handle({ options, logger, importUserLastLoggedAt = usecases.importUserLastLoggedAt }) {
+    const { file, dryRun } = options;
+
+    const total = file.length;
+    logger.info(`${total} users must be processed for last logged date update.`);
+    let count = 0;
```

Comment:
Ajouter une log pour dire que le mode dry run est activé. car actuellement quand on lit les logs retournée avec ou sans dry run il n'y a aucune différence.

```suggestion
    if (dryRun) logger.info('DRY RUN mode enabled - data updates are not effective');
    logger.info(`${total} users must be processed for last logged date update.`);
    let count = 0;
```